api/src/test/java/org/openmrs/module/fhir2/providers/r3/ConditionFhirR3ResourceProviderTest.java

```diff
@@ -212,4 +218,14 @@ public void searchConditions_shouldReturnConditionReturnedByServiceWhenPatientIs
 		assertThat(resultList.size(), greaterThanOrEqualTo(1));
 		assertThat(resultList.iterator().next().fhirType(), equalTo(FhirConstants.CONDITION));
 	}
```

Comment:
@sherrif10  add unit test for update methods.
Also failure in deletion as well

Reply:
ok thanks @Akayeshmantha , will add them

Reply:
Finished that